packages/preferences/src/browser/folders-preferences-provider.ts

```diff
@@ -21,87 +21,113 @@ import { FolderPreferenceProvider, FolderPreferenceProviderFactory } from './fol
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common';
 import URI from '@theia/core/lib/common/uri';
 
+export const LAUNCH_PROPERTY_NAME = 'launch';
+export type ResourceKind = 'settings' | 'launch';
```

Comment:
please use unique naming, see naming conventions

i.e. `FolderPreferenceKind``


`Resource` already has a meaning in Theia, i.e git and file are kind of resources